CHANGELOG.md

```diff
@@ -4,17 +4,15 @@
 
 ### Deprecations and removals
 
-* **The install_agent plugin has been officially removed.** The `install_agent` plugin was
-  deprecated in version 1.35 in favor of the `puppet_agent` plugin, and is now removed. The plugins
-  have the exact same behavior.
+* **The install_agent plugin has been officially removed.** The `install_agent` plugin was deprecated in version 1.35 in favor of the `puppet_agent` plugin, and is now removed. The plugins have the exact same behavior.
+
+* **Support for plan method `Target.new(<uri>, <options>)` will be dropped in Bolt 2.0.** Use `Target.new(<config>)`, where `config` is a hash with the same structure used to define targets in the inventory V2 file. See [the docs](https://puppet.com/docs/bolt/latest/writing_plans.html#creating-target-objects) for more information and examples.
 
 ### Bug fixes
 
 * **SSH commands will run from the home directory of the run-as user, not the connected user** ([#1518](https://github.com/puppetlabs/bolt/pull/1518))
 
-  Connecting via SSH and then switching users will now run as though it had
-  connected as the new user in the first place, using that user's home
-  directory as the working directory.
+  Connecting via SSH and then switching users will now run as though it had connected as the new user in the first place, using that user's home directory as the working directory.
```

Comment:
We (Heston, Alex, and I at least) prefer these stay broken over multiple lines. It makes git diffs easier to read and editing easier.